mesa/examples/basic/schelling/agents.py

```diff
@@ -20,11 +20,19 @@ def step(self) -> None:
             self.pos, moore=True, radius=self.model.radius
         )
 
+        type_of_neighbors = [neighbor.type == self.type for neighbor in neighbors]
+
         # Count similar neighbors
-        similar = sum(neighbor.type == self.type for neighbor in neighbors)
+        similar = sum(type_of_neighbors)
+
+        # Count total neighbors
+        total_neighbors = len(type_of_neighbors)
```

Comment:
these numbers will always be the same so this PR does not solve the problem.

Reply:
![image](https://github.com/user-attachments/assets/06302ce6-b1b1-409f-b27d-7f9ccda6f815)

I am getting these values while printing.